src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+            int supplyTotal = 0;
+            int buyTotal = 0;
```

Comment:
<img width="1025" alt="image" src="https://github.com/mate-academy/jv-work-with-file/assets/81781479/badbc7dc-881c-4292-9803-e64d49fc30f9">

you should divide logic into a few methods